### PR TITLE
LightInject as default container

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

**Comment by danielmarbach** [comment]
All green now by renaming LightInject.cs to LightInject.g.cs

File: src/NServiceBus.Core/ObjectBuilder/LightInject/LightInject.g.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@
 #endif
 
 #if NETCOREAPP2_0
-#define NETSTANDARD13
```
Comment (danielmarbach) [inline src/NServiceBus.Core/ObjectBuilder/LightInject/LightInject.g.cs]: Why was this change necessary?

File: src/NServiceBus.Core/ObjectBuilder/LightInject.g.cs [hunk 2]
```diff
@@ -26,10 +26,11 @@
     http://twitter.com/bernhardrichter
 ******************************************************************************/
 
-// how are the conditional compilation symbols set?
```
Comment (danielmarbach) [inline src/NServiceBus.Core/ObjectBuilder/LightInject.g.cs]: I don't see why this change was necessary

Comment (bording) [inline src/NServiceBus.Core/ObjectBuilder/LightInject.g.cs]: The comment I changed was added by @timbussmann when he wasn't sure how to handle the defines. Since what he had there was the best way to handle it, I changed the comment to indicate that vs it being a question.

File: src/NServiceBus.Core/ObjectBuilder/LightInject/LightInject.g.cs (hunk 1 above)
Comment (bording) [inline src/NServiceBus.Core/ObjectBuilder/LightInject/LightInject.g.cs]: These defines were added by @timbussmann. I changed it because netstandard1.6 is a better match for netcoreapp2.0 than netstandard1.3.

For example, if I had a netcoreapp2.0 project add added a NuGet package to it that had both 1.3 and 1.6 assemblies in it, the 1.6 one would be picked.
